integrations/selinux/teleport.fc

```diff
@@ -0,0 +1,4 @@
+/usr/local/bin/teleport  -- gen_context(system_u:object_r:teleport_exec_t,s0)
```

Comment:
This path now contains a symlink to the Teleport bin in all new systems, which does not match the `--` file type specified here.

On every new installation, Teleport now lives in `/opt/teleport/<installation-namespace>/versions/<teleport-version>(_ent)/bin/teleport`. The default installation namespace is `default`.

Reply:
As far as I can tell, the `--` file type works fine on symlinks, it's only supposed to be omitted for dirs. The current module works fine with it as is. Have you experienced something that leads you to believe otherwise?

Reply:
I think `--` does not select symlinks, there's a dedicated file mode for this, but I can't find it because how bad the selinux docs are. I think it's not an issue because executing the `/usr/local/bin/teleport` symlink got resolved, so we fall under the `/opt/teleport/...` context.

This would be ana issue if Teleport were to try to manage the symlink itself, which I think is not the case.

Reply:
Yeah I tried to find supporting docs too, was only able to find some just now. It seems `--` matches regular files only, omitting a file type matches all file types, `-d` matches directories specifically and `-l` matches symlinks. I think it's fine to leave as is, in case a deb or rpm package installs Teleport without `teleport-update` creating symlinks, `/usr/local/bin/teleport` will be a regular file right?

Docs here: https://github.com/SELinuxProject/selinux-notebook/blob/main/src/file_labeling_statements.md#genfscon